english/mad/mad.md

```diff
@@ -2,4 +2,4 @@ come along, I will introduce you the [Forest](../forest/forest.md)
 
 On second though, maybe I should [go back](../marshmallow.md).
 
-An army of cats appears! You stop to [Aaawww](https://www.reddit.com/r/cats/)
\ No newline at end of file
+An army of [cats](../follow-the-cat/follow-the-cat.md) appears! You stop to [Aaawww](https://www.reddit.com/r/cats/)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.